### DISCUSSION TITLE
I can't upload my profile image and Brand Logo

### DISCUSSION THREAD

**Discussion by min9538p** [outline/outline#7962] [Self Hosting]
When changing the profile photo or logo image, a message will appear indicating that the change was successful, but the change will not actually occur.

When attaching an image to a document, it operates normally.

Outline - Authenfik - redis - Minio - PostgreSQL is being used.

**Comment by min9538p** [comment]
The properly uploaded logo image and profile image can also be confirmed in minio.

**Reply by Con-JFS** [reply]
Did you get it working?

I always get the following http-response:
ApiClient.ts:104 POST http://outline:3000/api/teams.update 400 (Bad Request)
{
    "ok": false,
    "error": "validation_error",
    "status": 400,
    "message": "Must be a URL or relative path (avatarUrl)"
}
